src/moduler/feilmelding/FeilmeldingDetaljer.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { FeilmeldingType } from './FeilmeldingTypes';
+import styles from './FeilmeldingDetaljer.module.less';
+
+interface PropTypes {
+    feil: FeilmeldingType;
+}
+
+function DivIfPresent(props: { children?: React.ReactNode }) {
+    if (props.children === undefined || props.children === null) {
+        return null;
+    }
+    return <div>{props.children}</div>;
+}
+
+export default function FeilmeldingDetalj(props: PropTypes) {
+    const { melding, type, httpStatus, tekst } = props.feil;
+    const id: string = (melding && melding.id) || type;
+    const detaljer = melding && melding.detaljer;
+
+    return (
+        <div className={styles.feilmeldingdetaljer}>
+            <h2>{type}</h2>
+            <div>{id}</div>
+            <div>{httpStatus}</div>
+            <DivIfPresent>{detaljer?.detaljertType}</DivIfPresent>
+            <DivIfPresent>{detaljer?.detaljertType}</DivIfPresent>
+            <DivIfPresent>{detaljer?.feilMelding}</DivIfPresent>
+            {detaljer ? <pre>{detaljer.stackTrace}</pre> : null}
+            <DivIfPresent> {tekst} </DivIfPresent>
```

Comment:
noen steder har du space i ReactNodene dine  `<DivIfPresent> {tekst} </DivIfPresent>` andre ikke. her er det bedre å være konsistent

Reply:
Har litt lyst til at vi skal se på hva vi synes er ryddigst for sånne komponenter sammen. / kondisjonal reders

Reply:
Jeg er åpen for forslag. DivIfPresent ga mer mening fordi den skulle benyttes flere steder men ser ikke poenget med å flytte ut en så liten kodesnutt når det ikke er potensiale for gjenbruk.